onnx/backend/test/cpp/driver/test_driver.cc

```diff
@@ -199,17 +199,19 @@ ResolvedTestCase LoadSingleTestCase(const UnsolvedTestCase& t) {
   st.test_case_name_ = t.test_case_name_;
   ONNX_NAMESPACE::ParseProtoFromBytes(
       &st.model_, raw_model.c_str(), raw_model.size());
-
+  ONNX_NAMESPACE::ValueInfoProto input = &st.model_.graph.input;
```

Comment:
If I remember right, "graph.input" is a sequence of inputs. We will have to use the i-th element of this sequence when we process the input-file corresponding to the i-th input down below.